tests/functional/detection_status/test_detection_status.py

```diff
@@ -12,6 +12,7 @@
 
 import shared
 
+from codeCheckerDBAccess.ttypes import Encoding
```

Comment:
Please keep import order according to [CONTRIBUTING.md](https://github.com/Ericsson/codechecker/blob/master/CONTRIBUTING.md).

```py
import json
import os
import unittest

import shared
from codeCheckerDBAccess.ttypes import Encoding

from libtest import codechecer
from libtest import env
```